pkg/logger/logger.go

```diff
@@ -0,0 +1,66 @@
+package logger
+
+import (
+	cc "github.com/smartcontractkit/chainlink-common/pkg/logger"
+
+	"github.com/smartcontractkit/libocr/commontypes"
+)
+
+// conforms to the logger interface in chainlink-common
+var _ cc.Logger = Logger{}
+
+func appendArgs(lggr cc.Logger, newArgs ...interface{}) Logger {
+	var args []interface{}
+
+	switch t := lggr.(type) {
+	case Logger:
+		args = t.args
+		lggr = t.Logger
+	}
```

Comment:
There's some funny business in chainlink-common. It uses reflection to find `With` and `Named` in the `chainlink` implementation. In order to re-wrap the logger, we have to avoid nesting.

I think thats what happens anyway.